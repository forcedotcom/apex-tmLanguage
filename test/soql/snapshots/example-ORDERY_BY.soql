SELECT Name FROM Account ORDER BY Name DESC NULLS LAST
