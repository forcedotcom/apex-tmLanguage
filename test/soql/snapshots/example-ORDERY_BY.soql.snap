>SELECT Name FROM Account ORDER BY Name DESC NULLS LAST
#^^^^^^ source.soql keyword.operator.query.select.apex
#      ^ source.soql
#       ^^^^ source.soql keyword.query.field.apex
#           ^ source.soql
#            ^^^^ source.soql keyword.operator.query.from.apex
#                ^ source.soql
#                 ^^^^^^^ source.soql storage.type.apex
#                        ^ source.soql
#                         ^^^^^^^^ source.soql keyword.operator.query.orderby.apex
#                                 ^ source.soql
#                                  ^^^^ source.soql keyword.query.field.apex
#                                      ^ source.soql
#                                       ^^^^ source.soql keyword.operator.query.descending.apex
#                                           ^ source.soql
#                                            ^^^^^^^^^^ source.soql keyword.operator.query.nullslast.apex

>